patches/content-browser-webauth-webauth_request_security_checker.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/content/browser/webauth/webauth_request_security_checker.h b/content/browser/webauth/webauth_request_security_checker.h
+index 017f2e28a100b911cf8aa321b2395adf14925420..6447f4184890e8406d2fe4740407d3833f9698cd 100644
+--- a/content/browser/webauth/webauth_request_security_checker.h
++++ b/content/browser/webauth/webauth_request_security_checker.h
+@@ -58,6 +58,7 @@ class CONTENT_EXPORT WebAuthRequestSecurityChecker
+     // the return value is destroyed then the fetch will be canceled and
+     // |callback| will never be invoked.
+     static std::unique_ptr<RemoteValidation> Create(
++        scoped_refptr<network::SharedURLLoaderFactory> shared_url_loader_factory,
```

Comment:
you can pass `RenderFrameHost` here as 3rd parameter. This way you don't need to forward `SharedURLLoaderFactory` in the header, and the patch in `.cc` will look like:
```diff
   return RemoteValidation::Create(caller_origin, relying_party_id,
+                                  render_frame_host_,
                                   std::move(callback));
```